core/src/main/java/com/sequenceiq/cloudbreak/reactor/api/event/cluster/upgrade/ClusterUpgradeRequest.java

```diff
@@ -7,9 +7,4 @@
     public ClusterUpgradeRequest(Long stackId) {
         super(stackId);
     }
-
-//    @Override
```

Comment:
Why is this commented out?

Reply:
it's not commented. the commented code is deleted